lib/base_service.js

```diff
@@ -390,9 +389,15 @@ class BaseService {
                 config.logging.name = name;
             }
 
-            // And the statsd client
-            if (!config.metrics.name) {
-                config.metrics.name = name;
+            // And metrics clients
+            if (!Array.isArray(config.metrics)) {
+                console.log('Deprecation warning: \'metrics\' configuration should be an array');
```

Comment:
console... brrrr... Unfortunately this is done way before we've set up the proper logger, so I guess it's ok..